apps/api/src/auth/auth.service.ts

```diff
@@ -48,7 +48,7 @@ export class AuthService {
       this.OTP_EXPIRY
     )
 
-    await this.resend.sendOtp(email, otp.code)
+    this.resend.sendOtp(email, otp.code)
```

Comment:
The file is now situated under `src/auth/service` folder. I forgot to make the push yesterday, so please make sure to do this change